lib/rules/capitalized-comments.js

```diff
@@ -33,17 +28,27 @@ const SCHEMA_BODY = {
     type: "object",
     properties: {
         ignorePattern: {
-            type: "string"
+            type: "string",
+            default: ""
         },
         ignoreInlineComments: {
-            type: "boolean"
+            type: "boolean",
+            default: false
         },
         ignoreConsecutiveComments: {
-            type: "boolean"
+            type: "boolean",
+            default: false
         }
     },
     additionalProperties: false
 };
+const DEFAULTS = Object.keys(SCHEMA_BODY.properties).reduce(
+    (obj, current) => {
+        obj[current] = SCHEMA_BODY.properties[current].default;
+        return obj;
+    },
+    {}
+);
```

Comment:
I was wondering whether `ajv.validate()` will merge the default values. if so, no need to call `cloneDeep()` in linter.js, and no need to call `defaults = Object.keys()...` here.

not sure it works, but it maybe worth a try.

Reply:
What's the specific meaning of "merge the default values"?

Reply:
something similar to` Object.assign()`:
```js
var obj = {};
ajv.validate(obj); // default values were meged to obj
```

Reply:
Sure, it will be changed, so we need to clone it to prevent changing user-provided object.